v0/destinations/ga/transform.js

```diff
@@ -929,7 +929,11 @@ function processSingleMessage(message, destination) {
           ? nameToEventMap[eventName].category
           : ConfigCategory.NON_ECOM;
         category.hitType = "event";
-        customParams.ni = 1;
+        customParams.ni =
+          message.properties !== undefined &&
+          message.properties.nonInteraction !== undefined
+            ? message.properties.nonInteraction
+            : 1;
```

Comment:
We can use conditional chaining here.

Reply:
conditional chaining is generating 1 for nonInteraction set as 0. Removed the first condition and used conditional chaining for that